pkg/controller/vpc_nat.go

```diff
@@ -3,12 +3,15 @@ package controller
 import (
 	"fmt"
 
-	"k8s.io/klog/v2"
-
 	"github.com/kubeovn/kube-ovn/pkg/util"
+	"k8s.io/klog/v2"
```

Comment:
Internal imports should be put at the end. Please revert this change.

Reply:
```go
import (
	"fmt"

	"k8s.io/klog/v2"

	"github.com/kubeovn/kube-ovn/pkg/util"
)
```

Like this? My IDE is configured in goimports "package" mode just like your linter seems to be, but it doesn't order imports like that.